streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -652,6 +662,28 @@ public static String consumerPrefix(final String consumerProp) {
         return CONSUMER_PREFIX + consumerProp;
     }
 
+    /**
+     * Prefix a property with {@link #RESTORE_CONSUMER_PREFIX}. This is used to differentiate restore consumer configs
+     * from stream {@link ConsumerConfig consumer configs}.
```

Comment:
Nit: it's also to distinguish from other client configs -- not just consumer configs.